web_src/less/_base.less

```diff
@@ -1575,9 +1575,11 @@ a.ui.label:hover {
   }
 }
 
+/* I am not sure why this is needed, if yes, we need a comment
```

Comment:
```suggestion
```

Reply:
With:
![Screenshot from 2022-01-06 19-40-55](https://user-images.githubusercontent.com/1824502/148441623-b784f655-f5b9-42ea-a65f-6e16257a9d34.png)


Without:

![Screenshot from 2022-01-06 19-40-48](https://user-images.githubusercontent.com/1824502/148441596-6f289d45-b48e-4090-82eb-528045893677.png)

Reply:
The column is too wide without this. Setting the width to 0 means that width is only as wide as necessary.